### PR TITLE
Add stop operation feature

### PR DESCRIPTION
**Pull request by dbadura**
issue:
https://github.com/kyma-incubator/reconciler/issues/309

### CODE REVIEW

File: openapi/external_api.yaml [hunk 1]
```diff
@@ -18,6 +18,40 @@ servers:
         default: "v1"
 
 paths:
+  /operations/{schedulingID}/{correlationID}/stop:
```
Comment (dbadura) [inline openapi/external_api.yaml:21]: Maybe `stop` -> `disable`

File: cmd/mothership/mothership/start/http.go [hunk 2]
```diff
@@ -534,6 +540,66 @@ func deleteCluster(o *Options, w http.ResponseWriter, r *http.Request) {
...
+	op, err := getOperationStatus(o, schedulingID, correlationID)
+	if err != nil {
+		server.SendHTTPError(w, http.StatusInternalServerError, &reconciler.HTTPErrorResponse{
+			Error: errors.Wrap(err, "Failed to get operation").Error(),
+		})
+		return
+	}
+
+	if op.State != model.OperationStateNew {
+		server.SendHTTPError(w, http.StatusPreconditionFailed, &reconciler.HTTPErrorResponse{
```
Comment (dbadura) [inline cmd/mothership/mothership/start/http.go:586]: I should use different status code

Comment (dbadura) [reply cmd/mothership/mothership/start/http.go:586]: https://en.wikipedia.org/wiki/HTTP_403
looks ok

**Review by m00g3n — APPROVED** [review]
LGTM

**Review by tobiscr — APPROVED** [review]

**Review by suleymanakbas91 — APPROVED** [review]

**Review by suleymanakbas91 — APPROVED** [review]

**Pull request merged**